### PR TITLE
fix: containerd failed to load plugin

### PR DESCRIPTION
**Pull request by sergelogvinov**
Solve warning message at boot time: failed to load plugin io.containerd.internal.v1.opt

This is part of PR #3975

## Acceptance

Please use the following checklist:

- [ ] you linked an issue (if applicable)
- [ ] you included tests (if applicable)
- [ ] you ran conformance (`make conformance`)
- [x] you formatted your code (`make fmt`)
- [x] you linted your code (`make lint`)
- [ ] you generated documentation (`make docs`)
- [ ] you ran unit-tests (`make unit-tests`)

> See `make help` for a description of the available targets.

---
This change is [Reviewable](https://reviewable.io/reviews/talos-systems/talos/4547)

### CODE REVIEW

**Comment by smira** [comment]
This PR seems to expose one issue which wasn't frequent, but basically etcd fails to fetch the image on bootstrap. I want to look into it a bit more.

**Comment by smira** [comment]
@sergelogvinov I did a bit of refactoring add added config for both system `containerd` running from the tmpfs to make sure it doesn't load the CRI plugin for example.

File: hack/containerd.toml [hunk 1]
```diff
@@ -1,12 +1,7 @@
 version = 2
 
-disabled_plugins = ["io.containerd.snapshotter.v1.aufs", "io.containerd.v1.zfs", "io.containerd.snapshotter.v1.zfs", "io.containerd.v1.devmapper", "io.containerd.snapshotter.v1.devmapper", "io.containerd.snapshotter.v1.btrfs"]
```
Comment (smira) [inline hack/containerd.toml:3]: in fact I moved `hack/containerd.toml` to `hack/cri-containerd.toml`, and created new `hack/containerd.toml`

**Comment by sergelogvinov** [comment]
i've totally forgot about https://github.com/talos-systems/pkgs/blob/master/containerd/pkg.yaml#L24

We build containerd without those plugins ```disabled_plugins```. 
That's why i've removed it here #3975

Sorry )

**Comment by smira** [comment]
> i've totally forgot about https://github.com/talos-systems/pkgs/blob/master/containerd/pkg.yaml#L24
> 
> We build containerd without those plugins `disabled_plugins`. That's why i've removed it here #3975
> 
> Sorry )

no problem, let me clean up disabled plugins, but anyways disabling cri for core containerd makes sense, same as `/opt`)

**Comment by smira** [comment]
updated removing disabled plugins from disabled_plugins ;)

File: hack/cri-containerd.toml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+version = 2
+
+disabled_plugins = []
```
Comment (smira) [inline hack/cri-containerd.toml:3]: all mentioned plugins are disabled in the containerd build, no need to disable them once again

**Review by smira — APPROVED** [review]

**Pull request merged**